src/Context/CachedContextFactory.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Roave\BetterReflection\Context;
```

Comment:
Yeah, it is confusing and I did experiment with better names before giving up. A context is a namespace and a list of imported namespaces and aliases. Perhaps `NamespaceContext` or `NamespaceEnvironment` or <something better here>